helix-core/src/main/java/org/apache/helix/task/TaskStateModel.java

```diff
@@ -283,6 +292,38 @@ public void reset() {
     }
   }
 
+  /**
+   * Loads Task and TaskFactory classes for command input
+   */
+  private boolean loadNewTask(String command) {
+    try {
+      // Read ZNRecord containing task definition information.
+      ZNRecord taskConfig = _manager.getHelixDataAccessor().getBaseDataAccessor()
+          .get("/" + _manager.getClusterName() + "/TASK_DEFINITION",
+              null, AccessOption.THROW_EXCEPTION_IFNOTEXIST);
+
+      // Open the JAR file and import Task(s) and TaskFactory classes.
+      File taskJar = new File(taskConfig.getSimpleField("JAR_FILE"));
+      URL taskJarUrl = taskJar.toURI().toURL();
+
+      // Import Task(s) and TaskFactory classes.
+      URLClassLoader classLoader = URLClassLoader.newInstance(new URL[]{taskJarUrl});
+      for(String taskClass : taskConfig.getListField("TASK_CLASSES")) {
+        classLoader.loadClass(taskClass);
+      }
+      Class cl = classLoader.loadClass(taskConfig.getSimpleField("TASKFACTORY"));
```

Comment:
My suggestion here for you is to read about "magic strings" or magic numbers and understand why they could be bad. https://help.semmle.com/wiki/display/JAVA/Magic+strings%3A+use+defined+constant